astropy/io/misc/asdf/tags/coords/representation.py

```diff
@@ -0,0 +1,52 @@
+from importlib import import_module
+from asdf.yamlutil import custom_tree_to_tagged_tree
+
+from ...types import AstropyType
+
+
+REPRESENTATION_MODULE_WHITELIST = ['astropy.coordinates.representation',
+                                   'sunpy.coordinates.representation']
```

Comment:
Just curious, should sunpy register its own entry points to avoid mentions to sunpy here?

Reply:
It would be substantially more complex that way. This whitelist would basically have to become an entrypoint, which actually as I think about it makes it a slight security risk. If I could install `pwn.me` as a module and then read an asdf file with that module path in it would get executed. (Ignoring the fact that installing it in the first place is executing the code yada yada).